Library/Homebrew/cask/installer.rb

```diff
@@ -315,10 +309,13 @@ def missing_cask_and_formula_dependencies
       end
     end
 
-    def cask_and_formula_dependencies
+    # TODO: move dependencies to a separate class,
+    #       dependencies should also apply for `brew cask stage`,
+    #       override dependencies with `--force` or perhaps `--force-deps`
```

Comment:
Note to self: this comment should either get update or removed. `brew cask stage` is not longer a valid Homebrew command.

Reply:
Remove I reckon.

Reply:
```suggestion
```